### PR TITLE
Issue 46626: Show last 180 days in QC folders by default

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
To help focus on the most relevant data and help with some perf concerns, we want to change the default date range to the last 180 days of data

#### Changes
* Default to last 180 days
* For Last N Days ranges, consider them relative to the most recent sample in the folder, not today's date

### CODE REVIEW

File: webapp/TargetedMS/js/QCTrendPlotPanel.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
 
     calculateStartDateByOffset : function() {
         if (this.dateRangeOffset > 0) {
-            var todayMinusOffset = new Date();
+            var todayMinusOffset = this.maxAcquiredTime ? new Date(this.maxAcquiredTime) : new Date();
```
Comment (cnathe) [inline webapp/TargetedMS/js/QCTrendPlotPanel.js]: should probably rename this variable now: `startDateByOffset`?

File: src/org/labkey/targetedms/query/ReplicateManager.java [hunk 2]
```diff
@@ -52,6 +53,21 @@ public static SampleFile getSampleFile(long sampleFileId)
             .getObject(sampleFileId, SampleFile.class);
     }
 
+    public static SampleFile getNewestSampleFile(Container container)
```
Comment (cnathe) [inline src/org/labkey/targetedms/query/ReplicateManager.java]: where is this called? I am not seeing it used in this PR.

**Review by cnathe — CHANGES_REQUESTED** [review]
A couple of comments/questions. I will approve the PR after it is retargeted to 22.11 (once that is created today).

File: src/org/labkey/targetedms/query/ReplicateManager.java (hunk 2 above)
Comment (labkey-jeckels) [reply src/org/labkey/targetedms/query/ReplicateManager.java]: I removed the caller when I realized that the client already had the first/last dates available from an existing API call. I deleted this method.

File: webapp/TargetedMS/js/QCTrendPlotPanel.js (hunk 1 above)
Comment (labkey-jeckels) [reply webapp/TargetedMS/js/QCTrendPlotPanel.js]: Renamed

**Review by cnathe — APPROVED** [review]

**Pull request merged**